tests/bot/cogs/moderation/test_utils.py

```diff
@@ -41,54 +41,35 @@ def setUp(self):
         """
         test_cases = [
             {
-                "args": (self.ctx, self.member, "ban"),
                 "get_return_value": [],
                 "expected_output": False,
-                "send_params": None
+                "infraction_nr": None
             },
             {
-                "args": (self.ctx, self.member, "ban"),
-                "get_return_value": [{
-                    "id": 1,
-                    "inserted_at": "2018-11-22T07:24:06.132307Z",
-                    "expires_at": "5018-11-20T15:52:00Z",
-                    "active": True,
-                    "user": 1234,
-                    "actor": 1234,
-                    "type": "ban",
-                    "reason": "Test",
-                    "hidden": False
-                }],
+                "get_return_value": [{"id": 1}],
                 "expected_output": True,
-                "send_params": (
-                    f":x: According to my records, this user already has a ban infraction. "
-                    f"See infraction **#1**."
-                )
+                "infraction_nr": "**#1**"
```

Comment:
You could use the ID from `get_return_value` to get this value in the test, so including it here feels redundant. If you make the aforementioned ID some uncommon number like 83492, then you could forego having the `**` and `#` too i.e. it'll be safe to just search for the number directly.